[passport-gated] Passport migration (#1158)
snapshot-labs/snapshot-strategies: src/validations/passport-gated/README.md

# Gitcoin Passport Gated Validation

This repository provides a passport-gated validation strategy for Snapshot. The implementation integrates with the Gitcoin API to validate whether a user is authorized to vote on a proposal.

## Prerequisites

Before using this code, ensure that you have the following information stored in a `.env` file at the project root:

- `NEXT_PUBLIC_GC_API_KEY=<your-api-key>`

## Overview

> This implementation uses the Gitcoin Passport API to check whether a user has a valid passport by looking for their passport score.

Reviewer: ```suggestion
This implementation uses the Gitcoin Passport API to check whether a user has a valid passport by looking for their stamps.
```

> ## Code Explanation
> 
> The main function in this codebase returns a binary score based on whether the user's passport is flagged as a likely Sybil.
> 
> ## Modifications
> 
> The original code utilized the Passport SDK to check if the user has a valid passport. However, with the introduction of the Passport API, we can now simplify the process by checking for a score.
> 
> ## Last Modified
> 
> This code was last modified on May 11, 2023.
> 
> Feel free to customize and extend this implementation to suit your specific needs.

Reviewer: ```suggestion
## Code Explanation
[Soon] The main function in this codebase returns a binary score based on whether the user's passport is flagged as a likely Sybil.

## Modifications

The original code utilized the Passport SDK to check if the user has a valid passport. [Soon] However, with the introduction of the Passport API, we can now simplify the process by checking for a score.

## Last Modified

This code was last modified on June 1, 2023.

Feel free to customize and extend this implementation to suit your specific needs.
```

# Gitcoin Passport Gated Validation

This repository provides a passport-gated validation strategy for Snapshot. The implementation integrates with the Gitcoin API to validate whether a user is authorized to vote on a proposal.

## Prerequisites

Before using this code, ensure that you have the following information stored in a `.env` file at the project root:

- `NEXT_PUBLIC_GC_API_KEY=<your-api-key>`

> - `NEXT_PUBLIC_GC_SCORER_ID=<your-scorer-id>`

Reviewer 2: ```suggestion
```
